pages/post/new.vue

```diff
@@ -116,10 +123,16 @@ const onPost = async () => {
     imageFiles.value = [];
     previews.value = [];
     uploadProgress.value = [];
+    navigateTo('/home');
   } else {
     console.error(error.value);
   }
 };
+
+const navigateTo = (path: string) => {
+  useNuxtApp().$router.push(path);
+};
+
```

Comment:
Mega bruh. U must create a router object like 
const router = useRouter(), or smth like this. And then use it instead of $router and useNuxtApp

Please переделай